recipes/sqlalchemy-oso/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.26.1" %}
+
+package:
+  name: sqlalchemy-oso
+  version: {{ version }}
+
+source:
+  url: https://github.com/osohq/oso/archive/refs/tags/sqlalchemy-v{{ version }}.tar.gz
+  sha256: 121d1b420c17250d57253d84004dcb7c8087579436c86a60f701a139e4781064
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install languages/python/sqlalchemy-oso -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - cffi >=1.0.0
+  run:
+    - python >=3.6
+    - packaging ==20.9
+    # fails pip check: https://github.com/conda-forge/importlib_metadata-feedstock/issues/88
+    - importlib-metadata !=4.7.1
+    - cffi >=1.0.0
+    - oso >=0.26.1
+    - sqlalchemy >=1.3.17,<1.5
+    - flask-sqlalchemy
```

Comment:
I think because one of the tests imports [here](https://github.com/osohq/oso/blob/bc0c5b77e150df8fb273489525b1ff51fb53fe4b/languages/python/sqlalchemy-oso/tests/test_flask.py#L5) which raises an error [here](https://github.com/osohq/oso/blob/bc0c5b77e150df8fb273489525b1ff51fb53fe4b/languages/python/sqlalchemy-oso/sqlalchemy_oso/flask.py#L14)  before skipping the flask tests it has become a hard dependency? 

might be misunderstanding something here

Reply:
right. just test. the rule of thumb is libraries want as few, and as loosely-pinned, runtime dependencies as possible. full-on applications will have more, and sometimes much more-tightly-pinned, dependencies.